### PR TITLE
Update dingtalk-robot

### PR DESCRIPTION
**Pull request by FishOrBear**
ref:https://github.com/x-cold/dingtalk-robot

### CODE REVIEW

**Review by bangbang93 — APPROVED** [review]

**Review by MysteryBlokHed — COMMENTED** [review]
If you update the tests (dingtalk-robot-sender-tests.ts), then this PR can be merged without review from a DefinitelyTyped maintainer.

You could add a test case in the constructor of `ChatBot` to include the `secret` key:

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b14f7ea5538afa63c44c05b157baca7c5cb86771/types/dingtalk-robot-sender/dingtalk-robot-sender-tests.ts#L3-L6>

This could be changed to look like this:

```typescript
const bot = new ChatBot({
    baseUrl: "https://oapi.dingtalk.com/robot/send",
    accessToken: "some token",
    secret: "foo",
});
```

**Review by MysteryBlokHed — APPROVED** [review]

**Comment by MysteryBlokHed** [comment]
> @FishOrBear: Everything looks good here. I am ready to merge this PR (at 2af582b) on your behalf whenever you think it's ready.
> 
> If you'd like that to happen, please post a comment saying:
> 
> > Ready to merge
> 
> and I'll merge this PR almost instantly. Thanks for helping out! ❤️

The PR should be ready to merge whenever you want to 😄

**Comment by bangbang93** [comment]
Ready to merge

**Pull request merged**
